SPECS/mstflint/mstflint.spec

```diff
@@ -40,13 +60,42 @@ find . -type f -iname '*.cpp' -exec chmod a-x '{}' ';'
 %configure --enable-fw-mgr --enable-adb-generic-tools
 %make_build
 
+#Compile mstflint driver
+export EXTRA_CFLAGS='-DVERSION=\"%version\"'
+cd kernel
+set -- *
+mkdir source
+mv "$@" source/
+mkdir obj
+for flavor in %flavors_to_build; do
+  export K_BUILD=%{kernel_source $flavor}
```

Comment:
This looks fishy. We're passing an argument to a function, which doesn't do anything with it.